### ISSUE TITLE
Rainier_2U: powersupply2 and powersupply3 appears in Inventory

### ISSUE THREAD

**Issue by vishwabmc** [ibm-openbmc/dev#3182]
Rainer_2U has only 2 PS.. but 2 more power supplies appear on all systems.

            ├─/xyz/openbmc_project/inventory/system/chassis/motherboard/powersupply0
            ├─/xyz/openbmc_project/inventory/system/chassis/motherboard/powersupply1
            ├─/xyz/openbmc_project/inventory/system/chassis/motherboard/powersupply2
            ├─/xyz/openbmc_project/inventory/system/chassis/motherboard/powersupply3

fitconfig=conf@aspeed-bmc-ibm-rainier.dtb

**Comment by vishwabmc** [comment]
Assigned to @mzipse to assign it to right Engineer

**Comment by mzipse** [comment]
@bjwyman , I'm going to assign this to you to investigate.  @derekhoward55 , fyi.

**Comment by bjwyman** [comment]
I thought this would be handled by a change @anoo1 has up for review, but now I am not so sure that is the case:
https://gerrit.openbmc-project.xyz/c/openbmc/phosphor-power/+/44440

**Comment by bjwyman** [comment]
This ended up fixed after #2430, due to changing from gpio-keys and phosphor-gpio-presence to using libgpiod directly in phosphor-psu-monitor.

```
root@p10bmc:~# busctl tree xyz.openbmc_project.Inventory.Manager
������/xyz
  ������/xyz/openbmc_project
    ������/xyz/openbmc_project/inventory
      ������/xyz/openbmc_project/inventory/system
        ������/xyz/openbmc_project/inventory/system/chassis
          ������/xyz/openbmc_project/inventory/system/chassis/motherboard
...
            ������/xyz/openbmc_project/inventory/system/chassis/motherboard/pcieslot9
            ������/xyz/openbmc_project/inventory/system/chassis/motherboard/powersupply0
            ������/xyz/openbmc_project/inventory/system/chassis/motherboard/powersupply1
            ������/xyz/openbmc_project/inventory/system/chassis/motherboard/tod_battery
            ������/xyz/openbmc_project/inventory/system/chassis/motherboard/tpm_wilson
            ������/xyz/openbmc_project/inventory/system/chassis/motherboard/vdd_vrm0
            ������/xyz/openbmc_project/inventory/system/chassis/motherboard/vdd_vrm1
root@p10bmc:~# 
```
